### PR TITLE
Update the release process

### PR DESCRIPTION
**Pull request by dliappis**
and add troubleshooting tip.

### CODE REVIEW

**Comment by dliappis** [comment]
I didn't add a milestone for this on purpose as it's a very minor, internal, item.

File: RELEASE.md [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+The release process in Rally is automated as much as possible.
+It will automatically run all tests (including integration tests), build and upload all artifacts and update the changelog and the list of contributors.
 
-This will automatically run all tests (including integration tests), build and upload all artifacts and update the changelog and the list of contributors.
+### Example: suppose you want to release Rally 2.0.1
 
-It is recommended to run `make release-checks release_version=1.2.0 next_version=1.2.1` before the release. The script will check for the requirements mentioned in [Preconditions](#preconditions) and [the Initial setup](#initial_setup).
+1. Run release checks to validate requirements mentioned in [Preconditions](#preconditions) and [the Initial setup](#initial_setup) using:
+
+    `make release-checks release_version=2.0.1 next_version=2.0.1` before the release.
```
Comment (danielmitterdorfer) [inline RELEASE.md:10]: Is `next_version` intentionally also 2.0.1?

File: RELEASE.md [hunk 2]
```diff
@@ -1,10 +1,22 @@
 ### How to release Rally?
 
-The release process in Rally is automated as much as possible. Suppose you want to release Rally 1.2.0. Then just run `make release release_version=1.2.0 next_version=1.2.1`. The parameter `release_version` refers to the release that is being released and the parameter `next_version` refers to the next development version. 
+The release process in Rally is automated as much as possible.
+It will automatically run all tests (including integration tests), build and upload all artifacts and update the changelog and the list of contributors.
 
-This will automatically run all tests (including integration tests), build and upload all artifacts and update the changelog and the list of contributors.
+### Example: suppose you want to release Rally 2.0.1
```
Comment (danielmitterdorfer) [inline RELEASE.md:6]: Did you mean to say Rally 2.0.0? The example where the `release` target is invoked suggests that.

File: RELEASE.md (hunk 1 above)
Comment (dliappis) [reply RELEASE.md]: Typo, fixed in 4a0c1db03491289fecd772e5157079f2c8be5c8e

File: RELEASE.md (hunk 2 above)
Comment (dliappis) [reply RELEASE.md]: Typo fixed in 4a0c1db03491289fecd772e5157079f2c8be5c8e

**Comment by dliappis** [comment]
Thanks for the review.

**Pull request merged**